libs/shared/ui-elements/src/lib/prio-button/prio-button.component.ts

```diff
@@ -9,10 +11,20 @@ import { Component, OnInit, Input, Output, EventEmitter, HostBinding } from '@an
 
 export class PrioButtonComponent implements OnInit {
 
-  inputFocus = false;
+  @Input() style = 'solid' ;  //solid || outline || flat
+  @Input() type = 'btn-primary' ;  //btn-primary || btn-secondary || btn-danger, ...etc
+  @Input() size = 'normal' ;   //normal || small
+  @Input() isBlock = false ;  // if true: button take dislay as block
+  @Input() isIcon = false ;  // if true: turns button view to icon (also if no text exists) and it takes class btn-icon
+  @Input() label = '' ;    //buttontext
+  // @Input() iconShape = '';
+  @Input() loadingState = ClrLoadingState.DEFAULT ;  // loading state
+  @Output() Click = new EventEmitter();   //click event
```

Comment:
Buttons can handle clicking by default so leaving this to be decided in case of the actualy use as the button can be of `type='submit'` or other html types etc

Reply:
I do not get what you exactly want, what if it has a different type than submit?

Reply:
That's exactly what i mean , we don't have to provide a click event for the button